package.json

```diff
@@ -18,8 +18,11 @@
   "homepage": "https://github.com/99xt/contributors-shield#readme",
   "dependencies": {
     "async": "^2.1.4",
+    "chai": "^4.1.1",
+    "express": "^4.15.4",
     "jimp": "^0.2.27",
     "lodash": "^4.17.4",
-    "request": "^2.79.0"
+    "mocha": "^3.5.0",
```

Comment:
Can we make mocha a dev dependency? @Janakshi28 WDYT?

Reply:
Sure 👍

Reply:
I guess you can move chai as well

Reply:
What about express?

Reply:
express defines a simple DSL (domain specific language) for routing and handling incoming HTTP requests and uring the server testing phase, we will need a way to send HTTP requests, for that we can use request.